backends/cassandra.go

```diff
@@ -5,50 +5,94 @@ import (
 
 	"github.com/gocql/gocql"
 	"github.com/prebid/prebid-cache/config"
+	"github.com/prebid/prebid-cache/utils"
 	log "github.com/sirupsen/logrus"
 )
 
-// Cassandra Object use to implement backend interface
-type Cassandra struct {
+type CassandraDB interface {
```

Comment:
Why the name change from Cassandra to CassandraDB?

Reply:
To emphasize it is a storage? I find it more descriptive. Want me to change it?